crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -37,16 +51,20 @@ module bridge::committee {
     struct BridgeCommittee has store {
         // commitee pub key and weight
         members: VecMap<vector<u8>, CommitteeMember>,
-        // threshold for each message type
-        thresholds: VecMap<u8, u64>
+        // min stake threshold for each message type, values are voting power (percentage), 2DP
+        stake_thresholds_percentage: VecMap<u8, u64>,
+        // Committee member registrations for the next committee creation.
+        member_registrations: VecMap<address, CommitteeMemberRegistration>,
+        // Epoch when the current committee was updated
+        last_committee_update_epoch: u64,
```

Comment:
We will need a record of when the committee is created to verify the voting power is correct? This is mainly for verification/auditing and it might not be very useful for bridge operations. Also I think this will become useful when we support committee change in future version.

Reply:
plz add a comment to explain the purpose